nasa-gibs/worldview
# WV-2991: Add NOAA-20 and NOAA-21 Chlorophyll a and PAR

## Description

Fixes #WV-2911

Added 4 layers - NOAA-20/VIIRS Chlorophyll a and Photosynthetically Available Radiation; NOAA-21/VIIRS Chlorophyll a and Photosynthetically Available Radiation

## How To Test

1. Open with[ these URL parameters](http://localhost:3000/?v=-306.31206515434263,-174.88702252712568,205.98720159096257,126.9300387792572&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,VIIRS_NOAA21_Photosynthetically_Available_Radiation,VIIRS_NOAA20_Photosynthetically_Available_Radiation,VIIRS_NOAA21_Chlorophyll_a,VIIRS_NOAA20_Chlorophyll_a,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2024-04-15-T00%3A47%3A47Z)
2. Verify expected result - Verify that the layers show imagery, and descriptions are correct. (*note: I have reached out to the data provider because the NOAA-21 doi doesn't currently work)


@nasa-gibs/worldview
